src/snowflake/telemetry/_internal/opentelemetry/exporter/otlp/proto/common/trace_encoder.py

```diff
@@ -0,0 +1,14 @@
+from typing import Sequence
+
+from opentelemetry.exporter.otlp.proto.http.trace_exporter.encoder import _ProtobufEncoder
+from opentelemetry.proto.collector.trace.v1.trace_service_pb2 import ExportTraceServiceRequest as PB2ExportTraceServiceRequest
+from opentelemetry.sdk.trace import ReadableSpan
+
+
+def encode_spans(
```

Comment:
call this `encode_traces`.

Reply:
This method signature matches what is in the latest opentelemetry-exporter-otlp-proto-common module:

```python
# opentelemetry-python/exporter/opentelemetry-exporter-otlp-proto-common/src/opentelemetry/exporter/otlp/proto/common/_internal/trace_encoder/__init__.py
def encode_spans(
    sdk_spans: Sequence[ReadableSpan],
) -> PB2ExportTraceServiceRequest:
```